src/pulselistener/pulselistener/lib/web.py

```diff
@@ -15,20 +14,23 @@ class WebServer(object):
     '''
     WebServer used to receive hook
     '''
+    QUEUE_CODE_REVIEW = 'codereview:out'
 
     def __init__(self):
```

Comment:
It'd be nice if users of WebServer could define their own queue on which the webserver should post builds.
We have two options:
1) Each user creates a Bus and a WebServer, passes the Bus to the WebServer constructor, the WebServer posts the builds to the Bus.
2) There is a singleton WebServer, each user creates a Bus and passes it to the already created WebServer (`register_queue`), the WebServer posts the builds to all the registered Buses.

Maybe 1 is better, so each service (code review, risk analysis, etc.) has its own server. On Phabricator we'd have three separate build plans, one for each service, which send the request to the right web server.
What do you think?

Reply:
I setup the queue name as external to the webserver: the client application controls its name